bin/_helm.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -eu
+
+# Checks if target/helm exists and it installs it if it doesn't.
+# Use `install_helm true` to also install Tiller.
```

Comment:
`install_helm` without params fails with `bash: $1: unbound variable`, just confirming that's expected.

Reply:
Good catch, I though I had that covered :-(